config/initializers/secure_headers.rb

```diff
@@ -6,11 +6,12 @@
   config.x_download_options = 'noopen'
   config.x_permitted_cross_domain_policies = 'none'
 
+  form_action = Rails.env.development? ? ["'self'", 'https://*'] : ["'self'"]
```

Comment:
Looking at this again, I'm wondering if we should throw in a comment explaining why we do this